index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="container">
+        <div id="template1">
+            <div class="bar">
+                <p>9:41</p>
+                <img class="mobile_network" src="images/header/tower.png" alt="network_img">
+                <img class="mobile_wifi" src="images/header/Wifi.png" alt="wifi_logo">
+                <img class="mobile_battery" src="images/header/Battery.png" alt="battery">
+            </div>
+            <header>
+                <a href="#" class="logo">My Logo</a>
+                <nav class="Union">
+                <img src="images/header/Union.png" alt="bar">
+                </nav>
+            </header>
+            <section>
+                <div class="bg_image">
+                    <img class="background" src="images/header/header.png" alt="background_img">
+                </div>
```

Comment:
- [ ] Kindly remove this image here and use it as a background-image with CSS for this `section`. You can check [here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) when to use background-image instead of a generic `img` tag.

Reply:
i didn't get this thing, can you plz elaborate this thing.
you mean I have to remove <img> and do it in this manner-
 <div background-image:url(image.jpeg)>;

Reply:
can you plz tell me i'm little bit confused